src/components/table/table.tsx

```diff
@@ -31,6 +31,7 @@ export default function Table<T extends object>({
                   name={column.name}
                   columnID={column.id}
                   width={column.width}
+                  onSort={() => {}}
```

Comment:
What about changing line 28 to be `column.sortable && typeof sortParams.sort === 'function'`
This would avoid showing the arrow at all in case onSort was not provided instead of showing a non functioning arrow

Reply:
Got it, so if onSort was absent we'd render a non-sortable column instead of a sortable no-op column? Makes sense